src/test/java/md/miller1995/Dealership/repositories/CarRepositoryTest.java

```diff
@@ -0,0 +1,68 @@
+package md.miller1995.Dealership.repositories;
+
+import md.miller1995.Dealership.models.entities.CarEntity;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+
+import java.time.LocalDate;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+@DataJpaTest
+class CarRepositoryTest {
+
+    @Autowired
+    private CarRepository carRepository;
+    CarEntity carEntity;
+
+    @BeforeEach
+    void setUp() {
+
```

Comment:
There are too many empty rows in this class.

Reply:
- deleted empty rows